tests/epyccel/modules/multi_rank.py

```diff
@@ -24,6 +24,9 @@ def augsub_mixed_order(a : 'float[:,:]', b : 'float[:,:](order=F)'):
 def augdiv_mixed_order(a : 'float[:,:]', b : 'float[:,:](order=F)'):
     a[:] /= b
 
+def add_mixed_order_ellipsis(a : 'float[:,:]', b : 'float[:,:](order=F)'):
+    a[...] = a + b
```

Comment:
It is related to the example used in the PR description, we need a way to do an array assignment of any rank, so that we can use it in a template with types of multiple ranks

Reply:
Ah! Good idea. I'm not sure this is really related to this PR though, it is somewhat lost here. I would rather create a separate PR for this even if it is tiny. If it is already here though I guess we could keep it. It should definitely be mentioned in the CHANGELOG though

Reply:
[14ae4bb](https://github.com/pyccel/pyccel/pull/1728/commits/14ae4bb46de3dae9d090bdff8b2a9137e1840dd8)